src/index.js

```diff
@@ -1,4 +1,152 @@
+	$('.sys').appendChild(img);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.sys').appendChild(p);
+	}
+};
+
+const mainData = (data, image) => {
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.main-data').appendChild(img);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.main-data').appendChild(p);
+	}
+};
+
+const weather = (data, image) => {
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.weather').appendChild(img);
+	const heading = document.createElement('p');
+	heading.innerHTML = 'Weather';
+	$('.weather').appendChild(heading);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.weather').appendChild(p);
+	}
+};
+
+const clearNodes = () => {
+	$('.town-name').innerHTML = '';
+	$('.degrees').innerHTML = '';
+	$('.sunset').innerHTML = '';
+	$('.sys').innerHTML = '';
+	$('.main-data').innerHTML = '';
+	$('.weather').innerHTML = '';
+};
+
+const changeBg = () => {
+	const weatherData = JSON.parse(localStorage.getItem('city'));
+	const sky = weatherData.weather[0].main;
+	if (sky === 'Clear') {
+		$('body').style.backgroundImage = 'url(../dist/img/weather.jpg)';
+	} else if (sky === 'Snow') {
+		$('body').style.backgroundImage = 'url(../dist/img/snow.gif)';
+	} else if (sky === 'Rain') {
+		$('body').style.backgroundImage = 'url(../dist/img/rain.gif)';
+	} else if (sky === 'Clouds') {
+		$('body').style.backgroundImage = 'url(../dist/img/cloudy.gif)';
+	} else {
+		$('body').style.backgroundImage = 'url(../dist/img/weather.jpg)';
+	}
+};
+
+const getApi = () => {
+	const city = $('#city-name').value;
+	const country = $('#country').value;
+	return `https://api.openweathermap.org/data/2.5/weather?q=${city},${country}&APPID=19162a83776adb533575bcaa858e87a0`;
+};
+
+const display = () => {
+	clearNodes();
+	changeBg();
+	const weatherData = JSON.parse(localStorage.getItem('city'));
+	cityname(weatherData.name, weatherData.weather[0].icon, weatherData.timezone);
+	degrees(weatherData.main.temp, weatherData.weather[0].icon);
+	latitute(weatherData.coord.lon, weatherData.coord.lat);
+	sys(weatherData.sys, weatherData.weather[0].icon);
+	mainData(weatherData.main, weatherData.weather[0].icon);
+	weather(weatherData.weather[0], weatherData.weather[0].icon);
+};
+
+$('#submit').addEventListener('click', (e) => {
+	e.preventDefault();
+	const url = getApi();
+	fetch(url, { mode: 'cors' })
+		.then(response => response.json())
+		.then((response) => {
+			localStorage.setItem('city', JSON.stringify(response));
+			display();
+		}).catch(err => `Error Reading data  ${err}`);
```

Comment:
let's do better error handling to let the user know there is a problem. e.g when the city name does not exit.

Reply:
done